staging/src/k8s.io/kubectl/pkg/cmd/explain/explain.go

```diff
@@ -58,7 +58,10 @@ var (
 		kubectl explain pods.spec.containers
 		
 		# Get the documentation of resources in different format
-		kubectl explain deployment --output=plaintext-openapiv2`))
+		kubectl explain deployment --output=plaintext-openapiv2
+		
+	        # Get the documentation of resources with short flag for output format
```

Comment:
I don't think we need an example of how to use the short flag,

It should hopefully be clear to people if they choose to use the short flag instead of the full flag.

I suggest just leaving the example as it was.

Reply:
@brianpursley  Thank you for the review, I have added the change.